addons/portal/static/src/js/crash_manager.js

```diff
@@ -0,0 +1,50 @@
+odoo.define('portal.CrashManager', function (require) {
+"use strict";
+
+var core = require('web.core');
+var CrashManager = require('web.CrashManager').CrashManager;
+var Notification = require('web.Notification');
+var AbstractService = require('web.AbstractService');
+
+var _t = core._t;
+
+/**
+ * An extension of Notification Widget to render the notifications on the website.
+ * @override
+ */
+var CrashManagerNotification = Notification.extend({
```

Comment:
No need of this, modify the `Notification` class directly and use it. No need to preload the Notifaction xml in the backend either.